starter-code/public/scripts/articleView.js

```diff
@@ -131,6 +133,8 @@ articleView.initAdminPage = function() {
   // TODO: Call the Handlebars `.compile` function, which will return a function for you to use where needed.
   // Make sure you assign the result of your Handlebars.compile call to a variable called "template", since
   // we are then calling "template" on line 117.
+  var source = $('#stats').text();
+   var template = Handlebars.compile(source);
```

Comment:
This can be done with one line. Generally you also want to be conscious how much memory your app is using. For this case its unimportant, but it larger apps, you wouldnt want to waste memory for a single variable that is going to only be used once.